CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.19 FATAL_ERROR)
```

Comment:
Can you guard the code for CMake VERSION_GREATER_EQUAL 19 instead?

Reply:
can do